services/web/server/src/simcore_service_webserver/payments/_rpc.py

```diff
@@ -0,0 +1,80 @@
+""" PRC client-side for the RPC server at the payments service
```

Comment:
Should it not be like this?
```suggestion
""" RPC client-side for the RPC server at the payments service
```